python/paddle/v2/framework/tests/test_pad_op.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+import numpy as np
+from paddle.v2.framework.op import Operator
+from gradient_checker import GradientChecker, create_op
+from op_test_util import OpTestMeta
+
+
+class TestPadOp(unittest.TestCase):
+    __metaclass__ = OpTestMeta
+
+    def setUp(self):
+        self.type = "pad"
+        self.inputs = {'X': np.random.random((16, 16)).astype("float32"), }
+        self.attrs = {}
+        self.attrs['paddings'] = [(0, 1), (2, 3)]
+        self.attrs['pad_value'] = 0
+        self.outputs = {
+            'Out': np.pad(self.inputs['X'],
+                          self.attrs['paddings'],
+                          mode='constant',
+                          constant_values=0)
```

Comment:
add unit testing for other shape and padding value.

Reply:
Done.